fix: added spacing below the sponsors text

MathurAditya724 opened the pull request:
Resolved Issue #1016 

- [x] added some margin above each sponsor component to make them evenly spaced

styfle reviewed (commented):
Why is the git diff so big for adding margin?

MathurAditya724 commented:
Changes -
- [x] Added sponsors in an array, in place of hard coding them in a single component.
- [x] Added a loop to render them
- [x] Set the height to auto, as Vercel and Upstash logo have different heights
- [x] Added margin-top property for the spacing (5px)
- [x] Removed the fragements, as it was not getting used.

The diff is quite big cause I shifted the svg logo from the component into an array. I haven't changed the svg's.

styfle reviewed (approved):
Thanks!
